structuredefinitions/Extension-England-LocationExtension.xml

```diff
@@ -0,0 +1,120 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-LocationExtension" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/LocationExtension" />
+  <version value="1.0.3" />
```

Comment:
version value should be 1.0.0

Reply:
Not sure about this. Historically it was 1.0.3 from the original place it existed. So the question is whether we keep the old version number or we revert to 1.0.0 for the purpose of it now being added to NHSE IG. Looking at some CodeSystems that were added in the past that came from other places, that uses an old canonical URL, use 1.0.0, so maybe we can just put it to 1.0.0 here. But I think it might be best to make sure it won't affect the BARS team.

Reply:
I agree we should use the old version number to stop duplication. We cant have two different 1.0.0 resources. The resource version is also independent of the IG version, so this is not an issue.

Reply:
Naminder mentioned that he is happy for it to go back to 1.0.0 though

Reply:
why would we?